src/util.js

```diff
@@ -117,7 +117,7 @@ export function getTileJson(glSource, styleUrl, options = {}) {
         promise = Promise.resolve(
           Object.assign({}, glSource, {
             url: undefined,
-            tiles: normalizedSourceUrl,
+            tiles: glSource.type === 'raster' ? [normalizedSourceUrl] : normalizedSourceUrl,
```

Comment:
Shouldn't the ternary expression be something like
```js
Array.isArray(normalizedSourceUrl) ? normalizedSourceUrl : [normalizedSourceUrl]
```
?

Reply:
I think I don't get it why we should check if the normalizedSourceUrl is an array.

While testing the mapbox style where I had the issue with, it turned out that the implementation inside setupRasterLayer expects an array for the tiles. For setupVectorLayer it doesn't except an array.

Reply:
Ok, thanks for investigating that. I thought it always expects an array, which would be consistent. Then I'll make that consistent in a separate pull request.